packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,479 @@
+| menu              | _slot_                                  | undefined     | Sets custom dropdown menu                                                                |
+| focusMode         | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`         | Sets the focus behavior for the Breadcrumb.                                              |
+| maxDisplayedItems | number                                  | undefined     | If items count more than `maxDisplayedItems` only this number of items will be displayed |
+| overflowIndex     | number                                  | 1             | Index of item where BreadcrumbMenu should be shown                                       |
+| size              | `small`, `medium`, `large`              | `medium`      | Defines size of the Breadcrumb                                                           |
+
+### BreadcrumbItem
+
+#### Anatomy
+
+![visual anatomy of the BreadcrumbItem component](./assets/breadcrumb-item-anatomy.png)
+
+BreadcrumbItem can be:
+
+- Button
+- Link
+- Non-clickable content
+- Dropdown Menu
+- Icon (optional)
+
+It can contain a tooltip.
+
+#### DOM
+
+Non-clickable element
+
+```HTML
+<li>
+  {children}
+</li>
+```
+
+Link
+
+```HTML
+<li>
+  <a href="#">
+    {children}
+  </a>
+</li>
+```
+
+Button
+
+```HTML
+<li>
+  <button>
+    {children}
+  </button>
+</li>
+```
+
+Usage
+
+```jsx
+<BreadcrumbItem>
+  Item 1
+</BreadcrumbItem>
+<BreadcrumbItem onClick={() => console.log('smth...')}>
+  Item 2
+</BreadcrumbItem>
+<BreadcrumbItem href="#">
+  Item 3
+</BreadcrumbItem>
+```
+
+#### API
+
+| Property     | Values                                  | Default   | Purpose                                     |
+| ------------ | --------------------------------------- | --------- | ------------------------------------------- |
+| active       | boolean                                 | false     | Indicates if the item is active             |
+| disabled     | boolean                                 | false     | Disables Breadcrumb item                    |
+| divider      | _slot_                                  | `chevron` | Sets type of divider                        |
+| focusMode    | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`     | Sets the focus behavior for the Breadcrumb. |
+| icon         | _slot_                                  |           | Sets icon                                   |
+| iconPosition | `before`, `after`                       | `before`  | Sets icon position                          |
+| tooltip      | _slot_                                  | undefined | Sets custom tooltip                         |
+
+#### Breadcrumb icon
+
+An icon can be SVG or image.
+The icon might be added as a prop:
+
+```jsx
+<BreadcrumbItem icon={<IconComponent />}>Item 1</BreadcrumbItem>
+<BreadcrumbItem icon="../path/name.png">Item 1</BreadcrumbItem>
+<BreadcrumbItem>
+  <SomeSVG />
+  Item 1
+</BreadcrumbItem>
+```
+
+### BreadcrumbDivider
```

Comment:
Please add some code examples of how the BreadcrumbDivider should be used, I don't see it as a component or a slot right now, so it's kind of mysterious for me